libs/libarchfpga/src/read_xml_arch_file.cpp

```diff
@@ -3316,7 +3316,11 @@ bool check_leaf_pb_model_timing_consistency(const t_pb_type* pb_type, const t_ar
             break;
         }
     }
-    VTR_ASSERT(model != nullptr);
+    if (model == nullptr) {
+        archfpga_throw(get_arch_file_name(), -1,
```

Comment:
I'm unsure how to get a useful line number for the second argument so I'm just using -1 currently.

Reply:
Once we've moved to the internal data structures (i.e. no longer processing the XML DOM elements) then we no longer keep line information around.  Using -1 is OK in that case.